closeio/utils.py

```diff
@@ -160,6 +160,32 @@ def paginate(func, *args, **kwargs):
             skip += limit
 
 
+def paginate_via_cursor(func, *args, **kwargs):
```

Comment:
Shall we add a test for it?

Reply:
hmm... yes you're right, a test would be nice. I need to think about how I can test this properly without creating hundreds of event logs in the test. I'll think about it and will not merge it until then.